hudi-client/src/main/java/org/apache/hudi/HoodieWriteClient.java

```diff
@@ -509,13 +540,21 @@ private Partitioner getPartitioner(HoodieTable table, boolean isUpsert, Workload
 
   /**
    * Commit changes performed at the given commitTime marker.
+   *
+   * @param commitTime Commit Time handle
```

Comment:
just `instant time of the commit` . . `Commit Time Handle` does not sound accurate to me.